tests/integration/general/test_asset_recording.py

```diff
@@ -38,8 +40,39 @@ class AssetRecordingBaseTestCase(integration.TestCase):
 
     def setUp(self):
         super().setUp()
+        # The combination of snapd, lxd and armhf does not currently work.
+        if os.environ.get("ADT_TEST") and self.deb_arch == "armhf":
+            self.skipTest("The autopkgtest armhf runners can't install snaps")
+
         self.useFixture(fixtures.EnvironmentVariable("SNAPCRAFT_BUILD_INFO", "1"))
 
+        # snapd has no easy way of telling me if a snap is installed from its cli.
+        try:
+            subprocess.check_output(["review-tools.snap-review", "--help"])
```

Comment:
`snap list review-tools` should exit 0 if it's installed, non-zero otherwise. That's probably better than relying on `--help` always being there in the review tools.